.swiftlint.yml

```diff
@@ -30,6 +30,8 @@ file_length:
 identifier_name:
   excluded:
     - id
+    - c
+    - r
```

Comment:
Tbh I am not a big fan of singe char names because it's not obvious when you read them.
If you work with more languages makes it even harder.
https://developer.apple.com/documentation/swift/array/reduce(_:_:)
they use `x, y`, but IMO the best would be `initial` `nextPartial` or something like that, so its obvious from a first glance